.github/scripts/spellcheck.words

```diff
@@ -42,6 +42,11 @@ Automake
 Autotools
 autotools
 AVR
+AWS
+AWS-LC
+AWSLC
+aws-lc
+awslc
```

Comment:
Do we really need to spell it this many different ways? Is there not an official way we could stick to?